kuma/static/styles/components/wiki/quick-links.scss

```diff
@@ -100,3 +100,8 @@
         }
     }
 }
+
+#wikiArticle .Quick_links,
+#wikiArticle #Quick_Links {
+    display: none;
+}
```

Comment:
I tested this one the following pages:

* http://localhost:8000/en-US/docs/Web/HTML/Element
* http://localhost:8000/en-US/docs/Web/JavaScript/Reference
* http://localhost:8000/en-US/docs/Mozilla/Add-ons/WebExtensions
* http://localhost:8000/en-US/docs/Web/API/Canvas_API/Tutorial
* http://localhost:8000/en-US/docs/Games

This hides the sidebars as intended. We can, and should, however simplify the above CSS to only:

```
.Quick_links {
    display: none;
}
```

Thanks @ExE-Boss

Reply:
Outside article preview the class for quick links is `quick-links`

Reply:
[The Kuma documentation says that it&nbsp;looks for an element with the&nbsp;id&nbsp;of&nbsp;`Quick_Links`.](https://developer.mozilla.org/docs/MDN/Contribute/Structures/Quicklinks#Quicklinks_syntax)

Reply:
If you look for example at the HTML/Element page that uses the HTMLSidebar:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element

You can see that the `id` and `class` is `quick-links`

<img width="502" alt="screenshot 2018-10-03 at 20 37 19" src="https://user-images.githubusercontent.com/10350960/46431803-0c71e480-c74d-11e8-8138-53078b31d1c9.png">

Now, if you go to edit and then preview, you will see that the `id` and `class` has changed to `Quick-Links` which is how it is defined in the macro

<img width="656" alt="screenshot 2018-10-03 at 20 36 14" src="https://user-images.githubusercontent.com/10350960/46431811-11369880-c74d-11e8-8ecd-696c94868474.png">

You will also notice that the JS in `wiki.js` references these using the `#quick-links` id:

https://github.com/mozilla/kuma/blob/master/kuma/static/js/wiki.js#L14

I believe this document macro plays a role in the change:
https://github.com/mozilla/kuma/blob/master/kuma/wiki/jinja2/wiki/includes/document_macros.html#L21

Also please feel free to test it locally with my suggestions or, using Chrome's local overrides.